subsys/bluetooth/shell/l2cap.c

```diff
@@ -311,7 +311,7 @@ static int cmd_ecred_reconfigure(const struct shell *sh, size_t argc, char *argv
 
 static int cmd_ecred_connect(const struct shell *sh, size_t argc, char *argv[])
 {
-	struct bt_l2cap_chan *l2cap_ecred_chans[1] = {&l2ch_chan.ch.chan};
+	struct bt_l2cap_chan *l2cap_ecred_chans[] = {&l2ch_chan.ch.chan, NULL};
```

Comment:
Good catch, but I guess you could fix the coding style in the same go, i.e. space after `{` and before `}`. Or does the same style issue exist in the entire file?

Reply:
I see now that the same bug exists for `ecred-reconfigure` as well. Will fix that and the style issue.